doc/user/content/ops/diagnosing-using-sql.md

```diff
@@ -189,7 +180,7 @@ where
 order by ratio desc;
 ```
 
-### I found a problematic operator! How do I find where it came from?
+### I found a problematic operator. Where did it come from?
 
 Look up the operator in `mz_dataflow_operator_addresses`. If an operator has
 value `x` in slot `n`, then it is part of the `x` subregion of the region
```

Comment:
This text maybe be out-dated at this point. @aljoscha changed the SLOT and VALUE stuff, but tagging him in to confirm as I haven't used it since the change.

Reply:
Yes, I changed this in the docs as well: https://github.com/MaterializeInc/materialize/blob/main/doc/user/content/ops/diagnosing-using-sql.md#i-found-a-problematic-operator-how-do-i-find-where-it-came-from